src/main/java/io/rcktapp/api/handler/util/LogHandler.java

```diff
@@ -110,7 +111,18 @@ else if (req.getParams().containsKey("tenantid"))
                   {
                      logParams.put("tenantId", tenantId);
                   }
-                  Long logId = (Long) Sql.insertMap(conn, logTable, logParams);
+
+                  Object logIdGeneric = Sql.insertMap(conn, logTable, logParams);
```

Comment:
Can this ever return null?

Reply:
There are 0 entries in the LogChange table that have LogId as null, but I don't see any reason why this couldn't programmatically be null, so I'll add another case for that in the logic here.